moderation/commands.go

```diff
@@ -878,19 +880,19 @@ var ModerationCommands = []*commands.YAGCommand{
 			}
 
 			if parsed.Switches["id"].Value != nil {
-				var warn []*WarningModel
-				err = common.GORM.Where("guild_id = ? AND id = ?", parsed.GuildData.GS.ID, parsed.Switches["id"].Int()).First(&warn).Error
-				if err != nil && err != gorm.ErrRecordNotFound {
+				id := parsed.Switches["id"].Int()
+				warning, err := models.FindModerationWarningG(parsed.Context(), id)
```

Comment:
The previous query filtered by both `guild_id` and `id`, but this turns out to be unnecessary as the `id` column is the sole PK of the table and so uniquely identifies the warning -- it is not per-guild.